databricks/koalas/frame.py

```diff
@@ -2757,6 +2757,100 @@ def astype(self, dtype) -> 'DataFrame':
             self._metadata.index_columns + list(map(lambda ser: ser._scol, results)))
         return DataFrame(sdf, self._metadata.copy())
 
+    def add_prefix(self, prefix):
+        """
+        Prefix labels with string `prefix`.
+
+        For Series, the row labels are prefixed.
+        For DataFrame, the column labels are prefixed.
+
+        Parameters
+        ----------
+        prefix : str
+           The string to add before each label.
+
+        Returns
+        -------
+        DataFrame
+           New DataFrame with updated labels.
+
+        See Also
+        --------
+        Series.add_suffix: Suffix row labels with string `suffix`.
+        DataFrame.add_suffix: Suffix column labels with string `suffix`.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'A': [1, 2, 3, 4],  'B': [3, 4, 5, 6]})
```

Comment:
This could cause a flakiness with Python 3.5. Could you explicitly add `columns=['A', 'B']`?

Reply:
nit: need only one space between `],` and `'B':`.